apps/drec-api/src/pods/issuer/issuer.service.ts

```diff
@@ -87,8 +87,15 @@ export class IssuerService {
       return;
     }
 
-    // Convert from W to kW
-    const totalReadValueKw = Math.round(totalReadValue * 10 ** -3);
+    const totalReadValueKw = await this.handleLeftoverReads(
+      group,
+      +totalReadValue,
```

Comment:
IMHO such casting is very ugly, but I don't insist on using `Number()` :D

Reply:
By the way, this shold be a number already, so why are you casting that?

Reply:
For some reason it comes as a string...and I got an error, that`s why I casted it

Reply:
Well, I would debug why it's string, especially that you control the code... 
And if it comes from controller, such mapping should be done on controller level, otherwise typescript is useless

Reply:
I think it`s posgress`s fault or typeorm which is anyway shitty. Because it comes directly from the db as a string.

Reply:
Done. Float type fixes it so removed the casting